data/autoyast_sle15/autoyast_bcache.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <email/>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    <reg_server>{{SCC_URL}}</reg_server>
+    <addons config:type="list">
+      <addon>
+        <name>sle-module-basesystem</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-server-applications</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-desktop-applications</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+    </addons>
+  </suse_register>
+  <bootloader>
+      <global>
+          <timeout config:type="integer">-1</timeout>
+      </global>
+  </bootloader>
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <keyboard>
```

Comment:
you can delete 'keyboard', 'language', 'ntp-client', 'networking', 'firewall', 'timezone' sections as they are default.

Reply:
i did removed all of them but it started breaking the installation. So i moved them all back. Again this is the minimal i see in other files too.

Reply:
that is expected, it is tricky to find the minimal one, other files that you could have checked are not really minimal. Check [mini profile](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_sle15/mini.xml)
I think we should not pull out 'networking', the rest should be fine to delete them.

Reply:
I removed some. you can check later VR

Reply:
Perfect, thanks!